examples/quick_start/openai_example_speculative.py

```diff
@@ -0,0 +1,12 @@
+from sglang import function, gen, set_default_backend, OpenAI
+@function
```

Comment:
```suggestion
@function(spec_tokens=512)
```
Could you make the length of speculation as an argument?

Reply:
added now

@function(spec_tokens=512)
def example(s):
...